administrator-manual/en/mail.rst

```diff
@@ -493,6 +495,8 @@ The Rspamd web UI:
 * tracks the history of recent messages,
 * allows training the Bayes filter by submitting a message from the web form.
 
+For more specific informations on Rspamd settings, please read the :ref:`rspamd-section` page.
```

Comment:
I'd reduce the whole "Rspamd web interface" section: just keep the title, first period (lines 474-483) and a link to the new dedicated page.

The existing contents can be moved under the new page